cli/attestation.go

```diff
@@ -624,6 +625,39 @@ func (cli *CLI) NewValidateAttestationValidationCmd() *cobra.Command {
 	return cmd
 }
 
+var newMeasurementFunc = igvmmeasure.NewIgvmMeasurement
```

Comment:
This is done for testing only. It indicates that the "measurement provider" should be a dependency for CLI struct, just like SDK and other clients. Also, the CLI struct should not be exported.